### DISCUSSION TITLE
opentrack 2022.3.0

### DISCUSSION THREAD

**Discussion by sthalik** [opentrack/opentrack#1500] [Announcements]
### Features

- tracker/neuralnet: add support for operating in higher resolutions (by @DaWelter; #1476).
- module dialogs can now displayed as tabs even in low resolutions.
- a vendor can ship profile presets with the installer as `modules/presets/*.ini`.  they'll be copied to the user's profile directory under Documents (but won't be copied again if they're deleted).
- add libusb installer for the ps3 eye open driver to `contrib/libusbK-inf-wizard`, to be used with [this guide](https://www.irtrack.pl/download/ps3-open-driver/) by weczi of IRTrack. this installation procedure is better than the one on the wiki, but hasn't been extensively tested.
- tracker/trackhat: implement LED tracking status indicator, add back the gain slider, and fix custom models.

### Fixes

- tracker/steamvr: update to newer SDK version (report bugs in issues section or the discussion thread below).
- tracker/wii: fix tracking not working when the window is minimized and occluded, and a memory leak.

<hr /><em>This discussion was created from the release <a href='https://github.com/opentrack/opentrack/releases/tag/opentrack-2022.3.0'>opentrack 2022.3.0</a>.</em>

**Comment by samkatakouzinos** [comment]


**Reply by sthalik** [reply]
There was a last-minute issue with the installer. Sorry for the confusion.

**Reply by samkatakouzinos** [reply]
I see. I deleted my inital comment. Hope this thread can go with it.

**Comment by maxgit92** [comment]
![3](https://user-images.githubusercontent.com/101340363/188543350-a142adec-4a0c-49b0-a7fd-26d0e1ec23f2.png)
![1](https://user-images.githubusercontent.com/101340363/188543355-0a2f42e3-0c51-40f7-91a9-08685956c29e.png)
![2](https://user-images.githubusercontent.com/101340363/188543356-3f4ad105-6ccd-4880-8e09-38ff3dc6e9de.png)

Hello, are these latest releases carrying a virus?

**Reply by samkatakouzinos** [reply]
@maxigalera92 may you share what tool is this you're using?

**Reply by sthalik** [reply]
There are always some false positives on Virustotal. For instance, I'm getting reports of 1/74 on libraries that are part of a clean Windows 10 installation. None of the more reliable antivirus software report opentrack as infected.

Disclaimer: I built the binaries on the release page myself.
